deployment/mesh-infra/argocd/projects/plat-app-services/sads-offline/conf.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: sads-offline
+  namespace: argocd
+spec:
+  destination:
+    namespace: default
+    server: https://kubernetes.default.svc
+  project: plat-app-services
+  source:
+    path: deployment/plat-app-services/sads-offline
+    repoURL: https://github.com/c0c0n3/kitt4sme.live
+    targetRevision: HEAD
+  syncPolicy:
+    automated:
+      selfHeal: false
+    retry:
+      backoff:
+        duration: 5s
+        factor: 2
+        maxDuration: 3m
+      limit: 5
+    syncOptions:
+    - CreateNamespace=true
```

Comment:
we should delete this file. in fact the overlay you have in `kustomization.yaml` generates this content for you from `mesh-infra/argocd/projects/base`